ibis-server/app/mdl/substitute.py

```diff
@@ -39,13 +41,26 @@ def substitute(self, sql: str, write: str | None = None) -> str:
     def _build_model_dict(models) -> dict:
         def key(model):
             table_ref = model["tableReference"]
+
+            if not table_ref.get("catalog") and not table_ref.get("schema"):
+                logger.debug(
+                    "Try to substitute a tableReference has empty catalog and empty schema"
+                )
```

Comment:
I think we don't need this log. 🤔 
The problem is that if the table reference has only the table name, the return key is `..table`. However, it should be `table`. So I think we can change here:
If the table reference is fully qualified, the key is `catalog.schema.table`.
If the table reference is partially qualified, the key is `schema.table`.
If the table reference is unqualified, the key is `table`.